apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -848,6 +848,23 @@ function handleUncaughtErrors( calypsoPort ) {
 	};
 }
 
+function handleEditorLoaded( calypsoPort ) {
+	const unsubscribe = subscribe( () => {
+		const isReady = select( 'core/editor' ).__unstableIsEditorReady();
+		if ( isReady ) {
+			requestAnimationFrame( () => {
```

Comment:
After a few experiments in my local dev, the combination of `__unstableIsEditorReady` + `requestAnimationFrame` seems to be reasonably close to when the editor is rendered, both for new posts and posts with existing content.